src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reversed = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (reversed[value] !== undefined) {
+      return null;
+    } else {
```

Comment:
else is not necessary here, but it does not really matter

Reply:
I fixed it. Thanks)